### PR TITLE
Egress Perimeter Controls GA

### PR DESCRIPTION
**Pull request by mdlinville**
Ref: https://cockroachlabs.slack.com/archives/C03GUEQ308N/p1683666753481909

## Previews
[releases/cloud.md](https://deploy-preview-16916--cockroachdb-docs.netlify.app/docs/releases/cloud.html)
[v22.2/use-cloud-storage.md](https://deploy-preview-16916--cockroachdb-docs.netlify.app/docs/v22.2/use-cloud-storage.html)
[v22.2/changefeed-sinks.md](https://deploy-preview-16916--cockroachdb-docs.netlify.app/docs/v22.2/changefeed-sinks.html)
[v22.2/backup-and-restore-overview.md](https://deploy-preview-16916--cockroachdb-docs.netlify.app/docs/v22.2/backup-and-restore-overview.html)
[v23.1/use-cloud-storage.md](https://deploy-preview-16916--cockroachdb-docs.netlify.app/docs/v23.1/use-cloud-storage.html)
[v23.1/changefeed-sinks.md](https://deploy-preview-16916--cockroachdb-docs.netlify.app/docs/v23.1/changefeed-sinks.html)
[v23.1/backup-and-restore-overview.md](https://deploy-preview-16916--cockroachdb-docs.netlify.app/docs/v23.1/backup-and-restore-overview.html)
[cockroachcloud/egress-perimeter-controls.md](https://deploy-preview-16916--cockroachdb-docs.netlify.app/docs/cockroachcloud/egress-perimeter-controls.html)
[v22.2/cockroachdb-feature-availability.md](https://deploy-preview-16916--cockroachdb-docs.netlify.app/docs/v22.2/cockroachdb-feature-availability.html)
[v23.1/cockroachdb-feature-availability.md](https://deploy-preview-16916--cockroachdb-docs.netlify.app/docs/v23.1/cockroachdb-feature-availability.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

**Review by abhinavg6 — APPROVED** [review]
LGTM

File: _includes/v22.2/misc/note-egress-perimeter-cdc-backup.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {{site.data.alerts.callout_info}}
-We recommend enabling Egress Perimeter Controls on {{ site.data.products.dedicated }} clusters to mitigate the risk of data exfiltration when accessing external resources, such as cloud storage for change data capture or backup and restore operations. See [Egress Perimeter Controls for CockroachDB Dedicated (Preview)](../cockroachcloud/egress-perimeter-controls.html) for detail and setup instructions.
-{{site.data.alerts.end}}
\ No newline at end of file
+We recommend enabling Egress Perimeter Controls on {{ site.data.products.dedicated }} clusters to mitigate the risk of data exfiltration when accessing external resources, such as cloud storage for change data capture or backup and restore operations. See [Egress Perimeter Controls](../cockroachcloud/egress-perimeter-controls.html) for detail and setup instructions.
```
Comment (taroface) [inline _includes/v22.2/misc/note-egress-perimeter-cdc-backup.md:2]: Are we generally doing this now? I can't remember.

```suggestion
Cockroach Labs recommends enabling Egress Perimeter Controls on {{ site.data.products.dedicated }} clusters to mitigate the risk of data exfiltration when accessing external resources, such as cloud storage for change data capture or backup and restore operations. See [Egress Perimeter Controls](../cockroachcloud/egress-perimeter-controls.html) for detail and setup instructions.
```

File: _includes/v23.1/misc/note-egress-perimeter-cdc-backup.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {{site.data.alerts.callout_info}}
-We recommend enabling Egress Perimeter Controls on {{ site.data.products.dedicated }} clusters to mitigate the risk of data exfiltration when accessing external resources, such as cloud storage for change data capture or backup and restore operations. See [Egress Perimeter Controls for CockroachDB Dedicated (Preview)](../cockroachcloud/egress-perimeter-controls.html) for detail and setup instructions.
-{{site.data.alerts.end}}
\ No newline at end of file
+We recommend enabling Egress Perimeter Controls on {{ site.data.products.dedicated }} clusters to mitigate the risk of data exfiltration when accessing external resources, such as cloud storage for change data capture or backup and restore operations. See [Egress Perimeter Controls](../cockroachcloud/egress-perimeter-controls.html) for detail and setup instructions.
```
Comment (taroface) [inline _includes/v23.1/misc/note-egress-perimeter-cdc-backup.md:2]:
```suggestion
Cockroach Labs recommends enabling Egress Perimeter Controls on {{ site.data.products.dedicated }} clusters to mitigate the risk of data exfiltration when accessing external resources, such as cloud storage for change data capture or backup and restore operations. See [Egress Perimeter Controls](../cockroachcloud/egress-perimeter-controls.html) for detail and setup instructions.
```

**Review by taroface — APPROVED** [review]
LGTM; one suggestion.

**Pull request merged**